.env.example

```diff
@@ -6,3 +6,5 @@ NODE_ENV=development
 REACT_APP_API_URL=
 REACT_APP_MODE=dev
 REACT_APP_EXPIRATION_GAP=180 # sec
+DISABLE_FEATURES=
```

Comment:
Should be `REACT_APP_DISABLE_FEATURES`. Since only environment variables which have the prefix `REACT_APP_` get populated onto the `process.env` object.